worlds/default.sdf

```diff
@@ -238,5 +238,9 @@
       <longitude_deg> 8.546163739800146</longitude_deg>
       <elevation>0</elevation>
     </spherical_coordinates>
+
+    <include>
+      <uri>model://arucotag</uri>
+    </include>
```

Comment:
Please create a separate world for the aruco tag. IMHO, the default world should be a minimalistic world where anybody can run. Specific worlds should be used for specific feature testing.